MpvRemote/app/src/main/java/miccah/laptopremote/MainActivity.java

```diff
@@ -39,6 +40,11 @@ protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
 
+        BackgroundToggleButton subtitles2 = (miccah.mpvremote.BackgroundToggleButton)
+                findViewById(R.id.subtitles2);
+        subtitles2.setChecked(PreferenceManager.getDefaultSharedPreferences(this)
+                .getBoolean("SUBTITLES", true));
+
```

Comment:
What is this code doing? Where is the `subtitles2` view defined?

Reply:
sorry it's a typo.

Reply:
No worries, I see it's getting the `SUBTITLES` preference here but I'm not seeing where it gets set.

Reply:
my bad, I got it into another branch by mistake